x-pack/plugins/ml/public/timeseriesexplorer/timeseriesexplorer_controller.js

```diff
@@ -568,6 +571,7 @@ module.controller('MlTimeSeriesExplorerController', function (
 
       // Ensure the forecast data will be shown if hidden previously.
       $scope.showForecast = true;
+      $scope.$applyAsync();
```

Comment:
Is this one needed? The block below either calls `$scope.refresh()` or adjusts the time range on the dashboard, which in turn then calls `refresh()`

Reply:
Thanks for pointing that out, I didn’t know `timefilter...` end ups calling `refresh()` too that’s why I put it in, I'll remove it!